src/styles/global.css

```diff
@@ -93,3 +94,8 @@ hr {
   border-color: transparent;
   border-bottom: none;
 }
+
+.contains-task-list {
```

Comment:
Ok, Zed decided to change all single quotes to double quotes, but real changes only on this line